infra-as-code/bicep/modules/logging/logging.bicep

```diff
@@ -39,11 +38,10 @@ param parLogAnalyticsWorkspaceLogRetentionInDays int = 365
   'Updates'
   'VMInsights'
 ])
-@sys.description('Solutions that will be added to the Log Analytics Workspace. Default: [AgentHealthAssessment, AntiMalware, AzureActivity, ChangeTracking, Security, SecurityInsights, ServiceMap, SQLAssessment, Updates, VMInsights]')
+@sys.description('Solutions that will be added to the Log Analytics Workspace. Default: [AgentHealthAssessment, AntiMalware, ChangeTracking, Security, SecurityInsights, ServiceMap, SQLAssessment, Updates, VMInsights]')
```

Comment:
Need to remove the `Default:` section in each of the parameters description in this PR.

When the parameter markdowns are generated it displays the default value underneath. Other modules to be verified for same parameter description `Default:` detail.